docs/download.md

```diff
@@ -84,19 +84,20 @@ CUDA 11.8 through CUDA 12.0.
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
 * Verify the signature for Scala 2.12 jar:
-    `gpg --verify rapids-4-spark_2.12-24.06.1.jar.asc rapids-4-spark_2.12-24.06.1.jar`
+    `gpg --verify rapids-4-spark_2.12-24.08.0.jar.asc rapids-4-spark_2.12-24.08.0.jar`
 * Verify the signature for Scala 2.13 jar:
-    `gpg --verify rapids-4-spark_2.13-24.06.1.jar.asc rapids-4-spark_2.13-24.06.1.jar`
+    `gpg --verify rapids-4-spark_2.13-24.08.0.jar.asc rapids-4-spark_2.13-24.08.0.jar`
 
 The output of signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
-* Improve support for Unity Catalog on Databricks
-* Added support for parse_url PATH
-* Added support for array_filter
-* Added support for Spark 3.4.3
+* Support timezones with daylight savings shifts
+* Improve metrics in Spark UI
+* Refact parquet decode microkernels and support load balancing RLE runs
+* Support dynamic scan filtering
+* Improve UCX shuffle 
```

Comment:
May want to add:
- Improve get_json performance

Reply:
added